File: .sdk/tm/ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { Openf1CarDataSDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await Openf1CarDataSDK.test()
    equal(null !== testsdk, true)
  })

})

File: lua/test/exists_test.lua
-- ProjectName SDK exists test

local sdk = require("openf1-car-data_sdk")

describe("Openf1CarDataSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
